nemo/collections/tts/models/fastpitch.py

```diff
@@ -217,6 +217,12 @@ def _setup_normalizer(self, cfg):
                 raise ImportError(
                     "`nemo_text_processing` not installed, see https://github.com/NVIDIA/NeMo-text-processing for more details"
                 )
+            # fmt: off
+            import pkg_resources
+            # safeguard to the version 0.1.8rc0 to avoid unexpected normalized outputs for Chinese. Will remove the guard once
+            # next-text-processing new release fixes the bug. Tracking bugfix in https://github.com/NVIDIA/NeMo-text-processing/issues/109.
+            pkg_resources.require("nemo_text_processing==0.1.8rc0")
```

Comment:
have you seen https://setuptools.pypa.io/en/latest/pkg_resources.html:
```
 Users should refrain from new usage of pkg_resources and should work to port to importlib-based solutions.
```
also maybe add a better message for users + installation instructions?

Reply:
good catch! we are thinking of just pin `nemo-text-process<=0.1.8rc0` to simplify the temporal solution